immuni_exposure_ingestion/monitoring/api.py

```diff
@@ -0,0 +1,26 @@
+from prometheus_client.metrics import Counter
+
+from immuni_common.monitoring.core import NAMESPACE, Subsystem
+
```

Comment:
I'd add a note on why it has been chosen to write dedicated metrics (e.g., "dummy" would have been meaningless for the already existent REQUESTS_LATENCY metric, same for "province").